data/startup/tables/tile.lua

```diff
@@ -227,6 +227,61 @@ TileAction = {
 			{x = 32216, y = 31838, z = 15}
 		}
 	},
+	-- Threatened Dreams Quest
+	[25024] = {
+		itemId = false,
+		itemPos = {
+			{x = 33200, y = 31892, z = 7},
+			{x = 33201, y = 31892, z = 7},
+			{x = 33201, y = 31893, z = 7},
+			{x = 33201, y = 31894, z = 7}
+		}
+	},
+	[25025] = {
+		itemId = false,
+		itemPos = {
+			{x = 33258, y = 32402, z = 7},
+			{x = 33257, y = 32402, z = 7},
+			{x = 33257, y = 32401, z = 7},
+			{x = 33259, y = 32402, z = 7}
+		}
+	},
+	[25026] = {
+		itemId = false,
+		itemPos = {
+			{x = 33262, y = 32378, z = 7},
+			{x = 33262, y = 32377, z = 7},
+			{x = 33263, y = 32377, z = 7},
+			{x = 33264, y = 32377, z = 7},
+			{x = 33262, y = 32379, z = 7},
+			{x = 33263, y = 32379, z = 7},
+			{x = 33264, y = 32379, z = 7}
+		}
+	},
+	[25027] = {
+		itemId = false,
+		itemPos = {
+			{x = 33257, y = 32315, z = 7},
+			{x = 33257, y = 32314, z = 7},
+			{x = 33259, y = 32314, z = 7},
+			{x = 33259, y = 32315, z = 7}
+		}
+	},
+	[25028] = {
+		itemId = false,
+		itemPos = {
+			{x = 33249, y = 32267, z = 6},
+			{x = 33249, y = 32268, z = 6},
+			{x = 33250, y = 32268, z = 6}
+		}
+	},
+	[25029] = {
+		itemId = 10840,
+		itemPos = {
+			{x = 33619, y = 32304, z = 9},
+			{x = 33591, y = 32321, z = 10}
+		}
+	},
 	-- Others actions
 	-- Threatened Dreams Quest
 	[50301] = {
```

Comment:
this AIDs from 50301 to 50305  should be removed, they are conflicting with the new ones